runtime/doc/usr_05.txt

```diff
@@ -387,6 +387,25 @@ Only files of this filetype will use the settings from the plugin.  The <name>
 part of the plugin file doesn't matter, you can use it to have several plugins
 for the same filetype.  Note that it must end in ".vim".
 
+THE PLUG COMMAND                                                    *:plug*
+
+:plug[!] {spec}	        Add the plugin bundle defined by {spec} to 'runtimepath'.
+			{spec} can be a full path, the name of a directory in
+			the plugins folder (defined as `~/.vim/plugged/` by
+			default, see 'bundledir'), or it can have the form
+			`PREFIX/DIRNAME`.
+			If [!] is given, source the *.vim files in the plugin/
+			folder within {spec}.
```

Comment:
This is not needed. Normal plugin managers handle this fine on their own (and they do *much more* then just sourcing files in `plugin/`: those files may define various events and at least VAM does its best to call them, also messing with `:filetype` (needed to make ftdetect file work AFAIR)).

Also VAM detects when such methods are needed, it does not require user to know that some startup stages were missed and thus it needs to do additional (or different) actions.